manifests/config.pp

```diff
@@ -25,4 +26,12 @@
       content => template('libvirt/qemu.conf'),
     }
   }
+
+  $_default_conf = merge({'start_libvirtd' => 'yes'}, $default_conf)
```

Comment:
I would prefer, to have it like qemu the qemu configuration file:
if $default_conf is not empty, then write, else do nothing.
eg. in debian, the start of libvirtd is already enabled. so I do not have to touch that file.